Dzmitry_Kurlenka/10.Kubernetes installation/ci_3.yml

```diff
@@ -0,0 +1,3116 @@
+
```

Comment:
looks like htlm file inside, not yaml, please check

Reply:
I fixed the ci_3 file,
Check It, please.